plugins/package-managers/cargo/src/main/kotlin/Cargo.kt

```diff
@@ -198,8 +201,8 @@ class Cargo(
 
         val scopes = setOfNotNull(
             getTransitiveDependencies(groupedDependencies[""], "dependencies"),
```

Comment:
Should `""` also be extracted to a constant for conistency?

Reply:
That will get removed in some more upcoming refactorings, so I left it as-is.